src/setupTest.jsx

```diff
@@ -67,6 +67,23 @@ jest.mock('@edx/frontend-platform/i18n', () => {
   };
 });
 
+/*
+When .env.test is removed, uncomment the env vars below and add any environment variables for testing with Jest
+
+Context: Snapshot is not currently not set up to be able to parse the environment variables in env.config.js
```

Comment:
```suggestion
Context: Snapshot is not currently not set up to be able to parse the environment variables in env.config.js
```

The word "not" is repeated to make this a double negative